Src/Plugins/Tests.Security/TestData/ReviewPotentiallySensitiveFiles/ExpectedOutputs/without_password.sarif

```diff
@@ -0,0 +1,69 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
```

Comment:
>$schema [](start = 3, length = 7)

Let's please encode the rule id in our test files to make things clear, so this file should be SEC1003.PfxCryptographicKeyfile_without_password.sarif. Yes? moving forward, need to push this standard consistently across all code bases.

Reply:
ok. Will do that in the following PRs

---
In reply to: [554471425](https://github.com/microsoft/sarif-pattern-matcher/pull/96#discussion_r554471425) [](ancestors = 554471425)